crates/router/src/connector/bitpay.rs

```diff
@@ -313,7 +316,7 @@ impl ConnectorIntegration<api::Capture, types::PaymentsCaptureData, types::Payme
     fn get_request_body(
         &self,
         _req: &types::PaymentsCaptureRouterData,
-    ) -> CustomResult<Option<String>, errors::ConnectorError> {
+    ) -> CustomResult<Option<types::RequestBody>, errors::ConnectorError> {
         Err(errors::ConnectorError::NotImplemented("get_request_body method".to_string()).into())
```

Comment:
Hey, @SanchithHegde , @lsampras can we make use of `&'static str` in case of such errors, which aren't dynamically generated, it could reduce heap allocation?
Not required as a part of this PR, could create a issue for such issues